sdk/tests/test_utils/test_working_server.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+
+import pytest
+from pytest_asyncio import fixture
+
+from eidolon_ai_client.client import Agent, Machine
+from eidolon_ai_sdk.system.resources.resources_base import load_resources
+from eidolon_ai_sdk.test_utils.machine import TestMachine
+from eidolon_ai_sdk.test_utils.server import serve_thread
+from eidolon_ai_sdk.test_utils.vcr import vcr_patch
```

Comment:
This file shows what tests will look life for people who are using the library.

We provide helper methods, but no magic. Everything needs to be explicitly defined, but should only be 4 fixtures. In the future we can consider providing these fixtures explicitly ootb, but I think seeing the sausage will give some comfort.

Reply:
4 was a fat-finger typo. 3 fixtures.

the 3 fixtures are what needs to be explicitly defined.